doc/org.eclipse.python4capella.doc/doc/user/general/Debug.textile

```diff
@@ -0,0 +1,28 @@
+h1. Debug a Pyhton script
+
+This section explains how you can debug your Python4Capella script.
+
+
+h2. Within Capella
+
+You will need to enable Development capabilities if it's not already done:
+ - select the Window > Preferences menu
+ - with the filter a the top left corner search for "capabilities"
+ - select the Capabilities node on the left
+ - select the Development capability to enable it
+ - click the Apply and Close button
+ 
+To add a breakpoint, you can double click on the left ruler of the Python editor. This will add a beakpoint on the selected instruction.
+
+You can now select your script and right click on it:
+ - select the Debug As / EASE Script menu
+
+If your script take some arguments, you can change the launch configuration by right clicking on your script:
+ - select the Debug As / EASE Script menu with the CTRL (COMMAND for Mac OS) key pressed.
+
+This will open the launch configuration for your script and you can edit it to add parameters for instance.
+
+To step you can use the tool bar at the top:
```

Comment:
step in?

Reply:
step in general: step over, step into, step return